dotnet/Vaas/Vaas.sln.DotSettings.user

```diff
@@ -0,0 +1,12 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
This file should not be committed. You can use https://github.com/github/gitignore/blob/main/VisualStudio.gitignore as .gitignore